### PR TITLE
*: change default collation to utf8_bin

### PR DESCRIPTION
**Pull request by coocood**
Currently we don't support case insensitive index, our duplication check is always case sensitive.

We should not use case insensitive collation as default collation.

### CODE REVIEW

File: executor/executor_test.go [hunk 1]
```diff
@@ -1074,8 +1074,8 @@ func (s *testSuite) TestTableScan(c *C) {
 	c.Assert(len(result.Rows()), GreaterEqual, 4)
 	tk.MustExec("use test")
 	tk.MustExec("create database mytest")
-	rowStr1 := fmt.Sprintf("%s %s %s %s %v", "def", "mysql", "utf8", "utf8_general_ci", nil)
-	rowStr2 := fmt.Sprintf("%s %s %s %s %v", "def", "mytest", "utf8", "utf8_general_ci", nil)
+	rowStr1 := fmt.Sprintf("%s %s %s %s %v", "def", "mysql", "utf8", "utf8_bin", nil)
```
Comment (zimulala) [inline executor/executor_test.go]: I tested MySQL for the following results: 
```
| def          | mysql              | latin1                     | latin1_swedish_ci      | NULL     |
```

Comment (coocood) [inline executor/executor_test.go]: We don't support `latin1_swedish_ci` collation.

**Comment by zimulala** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**